testing/resources/docker_hooks_repo/hooks.yaml

```diff
@@ -4,6 +4,12 @@
     language: docker
     files: \.txt$
 
+-   id: docker-hook-arg
+    name: Docker test hook
+    entry: echo -n
```

Comment:
I think the docker container in question only implements -n, and all that does is remove newline if present